packages/meditrak-server/src/tests/routes/GETSurveyGroups/assertSurveyGroupsPermissions.test.js

```diff
@@ -30,12 +37,10 @@ describe('Permissions checker for GETSurveyGroups', async () => {
   before(async () => {
     //Set up the survey groups and their surveys
     surveyGroup1 = await findOrCreateDummyRecord(models.surveyGroup, {
-      id: 'survey_group_1_test',
-      name: `Test survey group 1`,
+      name: 'Test survey group 1',
```

Comment:
Oh nice, I didn't even think of the fact you can just leave it off and the id will be generated!